src/Styleguide/Components/SelectedCareerAchievements.tsx

```diff
@@ -74,9 +74,10 @@ export class SelectedCareerAchievements extends React.Component<
     }
   }
 
-  renderInsight(insight) {
+  renderInsight(insight, key) {
     return (
       <ArtistInsight
+        key={key}
```

Comment:
#minor 🤔 (I'm not an expert on what React exactly uses this for, but I think `key` here is supposed to be a unique string among the siblings, which are the other artist insights. So rather than the `index`, we can maybe just use their id? In this case, the 'id' of these insights can just be `insight.type`?)

Reply:
We do use `index` frequently for this so this is certainly fine, was just a thought!